erpnext/hr/doctype/staffing_plan/test_staffing_plan.py

```diff
@@ -92,5 +92,5 @@ def make_company():
 	company.abbr = "_TC10"
 	company.parent_company = "_Test Company"
 	company.default_currency = "INR"
-	company.country = "India"
+	company.country = "Pakistan"
```

Comment:
Why is this change needed?

Reply:
@Mangesh-Khairnar 
https://github.com/frappe/erpnext/pull/18488/commits/0f072d7d40ff820ebd237a02b96e81f629661854